src/uu/factor/src/numeric.rs

```diff
@@ -172,55 +197,141 @@ impl Arithmetic for Montgomery {
     }
 }
 
+pub(crate) trait OverflowingAdd: Sized {
```

Comment:
Probably add a note to remove this once `num-traits` adds overflowing traits.

Reply:
Done